### DISCUSSION TITLE
FreshRSS support?

### DISCUSSION THREAD

**Discussion by BashBandito** [ReadYouApp/ReadYou#478] [Q&A] [answered]
When will FreshRSS support be added?

**Comment by Ashinch** [comment, marked as answer]
#536 

It's finally done.

**Reply by Ashinch** [reply]
> I just downloaded it both from github and f-droid and I still can't click FreshRSS in the options.

Please see #542, it is still in the testing stage.

**Reply by candidexmedia** [reply]
Added some instructions here because I got lost on how to add it: https://github.com/Ashinch/ReadYou/issues/677#issuecomment-2030861124
